js/moq/package.json

```diff
@@ -6,8 +6,8 @@
 	"license": "(MIT OR Apache-2.0)",
 	"repository": "github:kixelated/moq",
 	"exports": {
-		".": "./src/index.ts",
-		"./zod": "./src/zod.ts"
+		".": "./dist/index.js",
```

Comment:
Ideally we can avoid this change because it makes local testing (via link) more difficult.

Reply:
I was having difficulties getting the types for `HangPublish` in the UI without this change. Is there another way to do that? Maybe I can avoid using the types, but it'd make things easier to have access to them.

Reply:
I got around this requirement by using esbuild to do the typescript handling in Rollup.